### PR TITLE
tools/benchmark: support serializable range benchmark

### PR DESCRIPTION
**Pull request by xiang90**
On my local machine, serializable range is 10x faster than range.

### CODE REVIEW

File: tools/benchmark/cmd/range.go [hunk 1]
```diff
@@ -35,11 +35,13 @@ var rangeCmd = &cobra.Command{
 
 var (
 	rangeTotal int
+	rangeMode  string
 )
 
 func init() {
 	RootCmd.AddCommand(rangeCmd)
 	rangeCmd.Flags().IntVar(&rangeTotal, "total", 10000, "Total number of range requests")
+	rangeCmd.Flags().StringVar(&rangeMode, "mod", "l", "Linearizable(l) or Serializable(s)")
```
Comment (heyitsanthony) [inline tools/benchmark/cmd/range.go]: `mod` isn't very descriptive, maybe `sync`?

Comment (xiang90) [inline tools/benchmark/cmd/range.go]: consistency?

Comment (heyitsanthony) [inline tools/benchmark/cmd/range.go]: sure, better than `mod`

**Comment by xiang90** [comment]
@heyitsanthony Fixed.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**